curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comment-your-javascript-code.md

```diff
@@ -2,6 +2,7 @@
 id: bd7123c9c441eddfaeb4bdef
 title: Comment Your JavaScript Code
 challengeType: 1
+removeComments: false
```

Comment:
I *believe* This should be the only change that needs to be mirrored on the Espanol and Chinese versions of the file.